vernac/vernacentries.ml

```diff
@@ -1776,7 +1777,7 @@ let vernac_search ~atts s gopt r =
     (* if goal selector is given and wrong, then let exceptions be raised. *)
     | Some g -> snd (Pfedit.get_goal_context g) , Some g
   in
-  let get_pattern c = snd (intern_constr_pattern env c) in
+  let get_pattern c = snd (intern_constr_pattern env Evd.empty c) in
```

Comment:
At line 1775 keep the evar map from `Pfedit.get_goal_context` and at line 1776 use from_env.